src/content/en/android/trusted-web-activity/query-parameters/index.md

```diff
@@ -45,7 +45,7 @@ The wizard will use the start_url from the Web Manifest as default and will ask
 the value, giving developers the change to add extra parameters to the url used to start the
 Progressive Web App.
 
-![Showing the Bubblewrap CLI output](/web/updates/images/2020/03/llama-pack-init-custom-start-url.png)
+![Showing the Bubblewrap CLI output](/web/updates/images/2020/03/bubblewrap-init-custom-start-url.png)
```

Comment:
I ran `npm test` locally. This image address is invalid, as the file was renamed to `src/content/en/updates/images/2020/03/bubblewrap-pack-init-custom-start-url.png` - There's an extra `-pack` segment . We should probably rename the file to `src/content/en/updates/images/2020/03/bubblewrap-init-custom-start-url.png`

Reply:
Good catch! I didn't catch that during `npm test`...
Fix: https://github.com/google/WebFundamentals/pull/8732/commits/7ca6e1049032040f9e8315a9d8d47bd0df506241